src/tcp_connections.rs

```diff
@@ -30,6 +30,9 @@ pub type InTcpStream<T> = Receiver<T>;
 pub type TcpReader<T> = Receiver<T>;
 pub type TcpWriter<T> = OutTcpStream<T>;
 
+/// Magic used to identify a crust packet
+pub const CRUST_PACKET_HEADER : [u8; 4] = [0x63, 0x72, 0x75, 0x31];  // "cru1"
```

Comment:
So we could use this in our tracer protocol or provide a mechanism for decrypting to get this? I like the tracer idea as it means people can develop in any p2p protocol with any encryption or obfuscation and switch to tracer for test networks. As p2p is be nature distributed getting keys means try decrypt with every key, so a tracer is a neat idea. Crust then allows an automatic multi protocol NAT Traversing solution for p2p networks and handles and try as many protocols that it can in order of efficiency. So we would need a debug type flag to allow crust to go into force tracer mode.